src/Client.php

```diff
@@ -247,5 +247,12 @@ private function _call($method, $params = array())
     {
         return $this->__soapCall($method, $params);
     }
+        public function getInvoiceMarkup($invoiceId, $type)
```

Comment:
Can you move this function to be below `getThresholdAndUsageInfo` in order to localize this call to other similar autotask calls?